portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -63,4 +69,14 @@ private String getParameter(HttpServletRequest request, String name, String defa
     }
     return value;
   }
+
+  /**
+   * Stores user comments in a form of persistent storage.
+   */
+  private void storeComments(String name, String comment) {
+    Entity taskEntity = new Entity("Comment");
+    taskEntity.setProperty("name", name);
```

Comment:
Use const string for "comment", "name". Like this:

private static final COMMENT_TABLE_NAME = "Comment";

Reply:
Done.